types/index.d.ts

```diff
@@ -0,0 +1,55 @@
+declare module "@3d-dice/dice-box" {
+  type DiceboxConfig = {
+    assetPath?: string;
+    delay?: number;
+    enableShadows?: boolean;
+    gravity?: number;
+    id?: string;
+    offscreen?: boolean;
+    origin?: string;
+    spinForce?: number;
+    startingHeight?: number;
+    theme?: string;
+    throwForce?: number;
+    zoomLevel?: number;
+  };
+
+  type RollObject = {
+    modifier?: number;
+    qty: number;
+    sides: number;
+    theme?: string;
+  };
+
+  type RollNotation = string | string[] | RollObject | RollObject[];
+
+  type DieResult = {
+    groupId: number;
+    id: number;
+    result: number;
+    rollId: number;
+    sides: number;
+    theme: string;
+  };
+
+  type RollResult = {
+    id: number;
+    modifier: number;
+    rolls: DieResult[];
```

Comment:
The documentation says this DieResult[] is correct but this is not what we are seeing returned by the promise.